build.xml

```diff
@@ -586,6 +586,11 @@ return [
     'extends' =&gt; 'sandal',
 ];
 </echo>
+    <echo file="${contentdir}/content.phtml">&lt;?=$this->partial("content/$page.phtml", []);?&gt;
```

Comment:
Rather than copying the contents of the parent content.phtml file, maybe we should use:

`<?=$this->parentTemplate('content/content.phtml')?>`

That way, we're still testing the existing parent theme's template, and if it changes, we don't have to change anything here.

Reply:
Good idea! I was a bit confused on how it's used, but `include` seems to be the way.

Reply:
Yes, sorry for misspeaking about how to use it. It will be good to actually have an example of its use in the codebase somewhere now. :-)